controller/envs.go

```diff
@@ -64,3 +67,44 @@ func (c *Controller) SaveEnvsToFile(ctx context.Context) error {
 
 	return nil
 }
+
+func (c *Controller) UpdateEnvs(ctx context.Context, envs *entity.Envs) (*entity.Envs, error) {
+	projectCfg, err := c.cfg.GetProjectConfigs()
+	if err != nil {
+		return nil, err
+	}
+	if val, ok := projectCfg.LockedEnvsNames[projectCfg.Environment]; ok && val {
+		fmt.Println(ui.Bold(ui.RedText("Protected Environment Detected!").String()))
+		confirm, err := ui.PromptYesNo("Continue updating variables?")
+		if err != nil {
+			return nil, err
+		}
+		if !confirm {
+			return nil, err
+		}
+	}
+
+	project, err := c.GetProject(ctx, projectCfg.Project)
+	if err != nil {
+		return nil, err
+	}
+
+	pluginID := ""
+	for _, p := range project.Plugins {
+		if p.Name == "env" {
+			pluginID = p.ID
+		}
+	}
```

Comment:
Should we wrap this into a `getPlugins` function?

Reply:
Leaning towards _no_ for now, since it wouldn't save on any code. Also, might need both project and plugins at the same time in some places. Can factor out later if needed.

Reply:
SG